packaging/homebrew/README.md

```diff
@@ -0,0 +1,34 @@
+# Building a Homebrewbrew package of the CLI
```

Comment:
"Homebrewbrew"? 😛

Reply:
Yes! It's the new 'meta' version that brews Homebrews. 🤦